doc/docs/concepts/views.md

```diff
@@ -179,12 +190,14 @@ Using the ViewDefinition editor you can:
 
 ## Output Data Formats
 
-### Conversion to PostgreSQL
+Applying the FHIR ViewDefinition resources to the "base dwh" will generate materialized views which represent a "constrained" set of data to be used for downstream analytics applications (such as dashboards or reporting). 
```

Comment:
nit1: please fix the style, e.g., break the lines at 80 chars.

nit2: consider adding a reference to the config param for enabling this, i.e., [viewDefinitionsDir](https://github.com/google/fhir-data-pipes/blob/ffecd7c7bf23d86bee0c6dde0dd5d549038f769d/pipelines/controller/config/application.yaml#L166C3-L166C21)